public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -210,5 +217,14 @@ const getStyles = (theme: GrafanaTheme2, hasActions: boolean) => {
         zIndex: theme.zIndex.portal,
       },
     }),
+    sidebarContainer: css({
+      position: 'fixed',
+      height: `calc(100% - ${TOP_BAR_LEVEL_HEIGHT}px)`,
+      zIndex: 2,
+      right: 0,
+    }),
+    pageContainerMenuDockedExtensionSidebar: css({
+      maxWidth: `calc(100% - ${EXTENSION_SIDEBAR_WIDTH})`,
```

Comment:
think we can extend this in the future to make the sidepane pane resizable but for this first iteration we can use fixed width.